src/test/java/jp/ikedam/jenkins/plugins/updatesitesmanager/DescribedUpdateSiteJenkinsTest.java

```diff
@@ -172,7 +172,6 @@ public void testDoConfigure() throws Exception
         // duplicate id
         {
             WebClient wc = new WebClient();
-            wc.setPrintContentOnFailingStatusCode(false);

```

Comment:
new versions of jenkins don't use custom fork of webclient and this method is removed